rust/src/cfd.rs

```diff
@@ -141,6 +142,8 @@ pub async fn open(order: &Order) -> Result<()> {
 
     let mut connection = db::acquire().await?;
 
+    // TODO: Remove these weird wrapper types, should not be needed for what we are doing
```

Comment:
👍 is this possible? I tried without the wrapper types and cargo was complaining that e.g. i32 is not implementing the sqlx type trait. That's the reason I've added the weird wrapper types 😅.